App.tsx

```diff
@@ -1,13 +1,28 @@
 import React from "react";
 
-import Auth from "@navigation/Auth";
 import { NavigationContainer } from "@react-navigation/native";
+import { useAuthState } from "react-firebase-hooks/auth";
+
+import Auth from "@navigation/Auth";
+import RoleResolver from "@navigation/RoleResolver";
+
+import { ToastAndroid } from "react-native";
+
+import useEffectAfterMount from "@hooks/useEffectAfterMount";
+import useFirestoreState from "@hooks/useFirestoreState";
+import { auth } from "@config/firebase";
```

Comment:
put `import { ToastAndroid } from "react-native"` below other lib imports, and put `@navigation`, `@hooks`, and `@config` together